### ISSUE TITLE
Highlight activity

### ISSUE THREAD

**Issue by danielmlow** [ReproNim/reproschema-ui#250]
Can we highlight the activity the participant is currently in the drop down menu?

**Comment by sanuann** [comment]
@danielmlow do you mean on the sidebar?

**Comment by danielmlow** [comment]
Yep. So they know where they are throughout the whole protocol to us it's obvious because it says "Mental Health" and we see PHQ9 but they have no idea.

**Comment by danielmlow** [comment]
Yep. So they know where they are throughout the whole protocol to us it's
obvious because it says "Mental Health" and we see PHQ9 but they have no
idea.

**Comment by satra** [comment]
doesn't the sidebar circle highlight it when you answer questions? are you saying give the active activity a different color? do note that the sidebar is never visible on a phone by default (so most participants will not see it). while this would be a good thing, given the priorities i would suggest filing this as a feature request more than an urgent priority. i think async sending of data is likely more useful.

**Comment by sanuann** [comment]
> doesn't the sidebar circle highlight it when you answer questions? are you saying give the active activity a different color? do note that the sidebar is never visible on a phone by default (so most participants will not see it). while this would be a good thing, given the priorities i would suggest filing this as a feature request more than an urgent priority. i think async sending of data is likely more useful.

you are right, the sidebar circle does highlight.

**Comment by danielmlow** [comment]
The circle fills up as you go through the activity, but the activity is not highlighted. 
![image](https://user-images.githubusercontent.com/14787791/134937948-e5816cf2-5a3c-4ad2-a5f8-57f0c54807ed.png)

So maybe it could highlight in a different color like this if you're doing that activity (it might not be obvious to participants where they are in the study). 
![image](https://user-images.githubusercontent.com/14787791/134938148-94ce4f05-c5eb-4da2-9e8d-e6e0163f0337.png)

Maybe even more ideally a vertical progress bar on the side would indicate how far along you are in the protocol. 

As satra says, not urgent.
